editors/code/package.json

```diff
@@ -1,31 +1,33 @@
 {
     "name": "clarity-lsp",
-    "displayName": "clarity-lsp",
-    "description": "Language Server Protocol implementation for Clarity",
-    "preview": true,
+    "displayName": "Clarity LSP",
+    "description": "Language Server Protocol implementation for Blockstack's Clarity smart contracting language",
+    "publisher": "blockstack",
```

Comment:
If we're changing the publisher, I'd need an admin access to the visual studio marketplace account `blockstack`, so that I can update this repo with a new token - I have a Github action packaging this project and deploying to the marketplace.

Reply:
maybe Aaron can help? His extension is using the "blockstack" publisher